src/routes/components/icon/+page.svelte

```diff
@@ -45,3 +45,18 @@
     </QDocsSection>
   </div>
 </QDocs>
+
+<style lang="scss">
+  // @todo: remove as soon as we have .text-<color> classes etc.
```

Comment:
We already have `.text-<color>` and `<color>` classes, they're inside the `scss/theme` folder